pkg/tfbridge/diff.go

```diff
@@ -170,7 +170,7 @@ func makePropertyDiff(name, path string, v resource.PropertyValue, tfDiff shim.I
 		case v.IsObject():
 			// If this value has a diff and is considered computed by Terraform, the diff will be woefully incomplete. In
 			// this case, do not recurse into the array; instead, just use the count diff for the details.
-			if d := tfDiff.Attribute(name + ".%"); d == nil || !d.NewComputed {
```

Comment:
I think this got lost in the merge, very unfortunate. Again emphasizes the need for accurate tests..